examples/hybrid/define_tc_quicklook_profiles.jl

```diff
@@ -1,102 +1,133 @@
+import Dates
+
 import ClimaCore: Fields, InputOutput
 using Plots
 
-function plot_tc_profiles(folder, hdf5_filename)
+"""
+    get_main_branch_buildkite_path()
 
-    input_filename = joinpath(folder, hdf5_filename)
-    output_filename = joinpath(
-        folder,
-        "____________________________________final_profiles.png",
-    )
+Return the folder of the most recent buildkite run that contains a `ref_counter.jl` file,
+and therefore also contains hdf5 files with output data.
+"""
+function get_main_branch_buildkite_path()
```

Comment:
This looks similar to `self_reference_or_path`. I'm wondering if we should move it into the same file? (and maybe later rename / move it)

Reply:
yes, it does basically the same thing, except that it fetches the _most recent_ data on the highest ref_counter, as opposed to the oldest data on the highest ref_counter.
I can move it in there as long as it's okay to do `include(../../regression_tests/self_reference_or_path.jl` in the `examples/hybrid/driver.jl` file.

Reply:
> I can move it...

I think that's fine, we use

```julia
include(
        joinpath(
            @__DIR__,
            "..",
            "..",
            "regression_tests",
            "regression_tests.jl",
        ),
    )
```
above in the driver.

Reply:
done.

Reply:
One thing I just realized is that the data on the most recent ref counter is not guaranteed to exist, because we only save data to central if the ref counter has changed. Can we cycle back on how this works, @haakon-e?

Reply:
Ah, I see that `get_main_branch_buildkite_path` is no longer referenced